.envrc

```diff
@@ -0,0 +1,6 @@
+export POSTGRES_PORT=5432
+export POSTGRES_DATABASE=situation_room
+export POSTGRES_USERNAME=root
+export POSTGRES_PASSWORD=password
+export POSTGRES_HOSTNAME=locahost
+export PLUG_PORT=4000
```

Comment:
Dont commit `.envrc` directly.  These kinds of things should be in the `.envrc.sample` file as an example for people to set up on their own.

Reply:
100% - I see that I have neglected to append `.envrc` into `.gitignore`